### DISCUSSION TITLE
移除查看其他用户的关注列表只能查看前5页的限制

### DISCUSSION THREAD

**Discussion by SpecterShell** [the1812/Bilibili-Evolved#2152] [功能建议 / Ideas]
在B站网页版查看其他用户的关注列表最多只能查看前5页（查看自己的关注列表没有限制），即便修改网络请求也无法获取之后页数的数据。

但手机客户端就可以正常查看其他用户的全部关注。手机客户端的关注列表界面实际上是嵌套网页，可以在电脑浏览器内直接打开：
`https://space.bilibili.com/h5/follow?mid={被查看人的ID}&type=follow`
例如：
[https://space.bilibili.com/h5/follow?mid=2&type=follow](https://space.bilibili.com/h5/follow?mid=2&type=follow)

也许可以通过手机网页版调用的API来解决电脑网页版的问题。

**Comment by SpecterShell** [comment]
试了一下手机网页版的接口调用，发现这个方法还是只能查看前5页，但每次能获取50条关注信息，总共可以获取250条关注信息。而电脑网页版每次只能获取20条关注信息，总共也只能获取100条关注信息。

行吧😅

**Comment by yuegou233976** [comment]
问问，现在还有全部关注信息的获取方法吗

**Comment by the1812** [comment]
这是服务端的限制, 脚本做不了什么
